sklearn/tests/test_multiclass.py

```diff
@@ -291,6 +291,7 @@ def test_ovr_multiclass():
         assert_array_equal(y_pred, [0, 0, 1])
 
 
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
Let's update the test to not raise the warning in the first place as this test is not focused on `SV`C in particular but probabilistic classifier in general.

```suggestion
```